src/components/filtermenu/filtermenu.js

```diff
@@ -177,40 +182,40 @@ define(['require', 'dom', 'focusManager', 'dialogHelper', 'loading', 'apphost',
             focusManager.focus(newElem);
         }
     }
-
-    function onInputCommand(e) {
+    centerFocus(elem, horiz, on) {
+        import('scrollHelper').then(({ default: scrollHelper }) => {
+            var fn = on ? 'on' : 'off';
+            scrollHelper.centerFocus[fn](elem, horiz);
+        });
+    }
+    onInputCommand(e) {
         switch (e.detail.command) {
             case 'left':
-                moveCheckboxFocus(e.target, -1);
+                this.moveCheckboxFocus(e.target, -1);
                 e.preventDefault();
                 break;
             case 'right':
-                moveCheckboxFocus(e.target, 1);
+                this.moveCheckboxFocus(e.target, 1);
                 e.preventDefault();
                 break;
             default:
                 break;
         }
     }
-
-    function FilterMenu() {
-
-    }
-
-    function bindCheckboxInput(context, on) {
+    bindCheckboxInput(context, on) {
         var elems = context.querySelectorAll('.checkboxList-verticalwrap');
         for (var i = 0, length = elems.length; i < length; i++) {
             if (on) {
-                inputManager.on(elems[i], onInputCommand);
+                inputManager.on(elems[i], this.onInputCommand);
             } else {
-                inputManager.off(elems[i], onInputCommand);
+                inputManager.off(elems[i], this.onInputCommand);
             }
         }
     }
-
-    FilterMenu.prototype.show = function (options) {
+    static show(options) {
+        const self = new FilterMenu();
```

Comment:
`list.js` already instantiates FilterMenu, it's nicer to do it that way than to make this static and alias and instance, imo.